README.md

```diff
@@ -46,10 +46,9 @@ Installation of development environment
 * get go dependencies `go mod download`
 * install axolotl-web dependencies: `cd axolotl-web&&npm install`
 
-Translations
-------------
+When setting up for the first time and maybe occasionally later you need to update the browser list with your installed browsers. Change into the axolotl-web subfolder and run the following command:
 
-For information how to translate, please see [TRANSLATE.md](docs/TRANSLATE.md).
+`npx browserslist@latest --update-db`
```

Comment:
Idea. Could we not add this line to the `scripts` section in the axolotl-web package.json? As browserslist is already listed as a devDependency we then also additionally have a fixed version and reproducible outcomes.

The scripts section the place I would go to if I would want an overview of the available commands for an npm package.

Reply:
I will add it to a later merge request